aggregator/src/aggregation/decoder/witgen/types.rs

```diff
@@ -204,6 +207,8 @@ pub enum ZstdTag {
     ZstdBlockLstream,
     /// Beginning of sequence section.
     ZstdBlockSequenceHeader,
+    /// sequence bitstream for recovering instructions
+    ZstdBlockSequenceData,
```

Comment:
Do we need one more for `ZstdBlockSequenceFse` or something to decode the FSE tables?

Reply:
Renamed and adjusted.